Limelight/Stream/VideoDecoderRenderer.m

```diff
@@ -10,11 +10,22 @@
 #import "StreamView.h"
 
 #include "Limelight.h"
+#include "PlatformThreads.h"
+#include "LinkedBlockingQueue.h"
```

Comment:
These are moonlight-common-c internal APIs that aren't intended to be used by Moonlight clients. They don't have any API stability guarantees that the normal external APIs have.

We should use the native platform APIs for this.